components/infobox/commons/infobox_patch_custom.lua

```diff
@@ -0,0 +1,20 @@
+---
+-- @Liquipedia
+-- wiki=rocketleague
+-- page=Module:Infobox/Patch/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Patch = require('Module:Infobox/Patch')
+
+local CustomPatch = {}
+
+function CustomPatch.run(frame)
+	local patch = Patch(frame)
+	patch.addCustomCells = CustomPatch.addCustomCells
+	patch.getChronologyData = CustomPatch.getChronologyData
```

Comment:
Wouldn't this overwrite the default handling of getChronologyData with nothing (nil) causing it not to work?

Reply:
refresh